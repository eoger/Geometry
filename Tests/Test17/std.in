C cercle 52 52 1
LIST
LOAD a.txt
LIST
LOAD b.txt
LIST
LOAD c.txt
LIST
EXIT