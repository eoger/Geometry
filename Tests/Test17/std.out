OK
C cercle 52 52 1
ERR
C cercle 52 52 1
ERR
C cercle 52 52 1
OK
L abc 0 3 0 4
PL bcd 1 8 6 0 7 7
C cercle 52 52 1
C def 5 5 5
R efg 8 8 10 10
OA poa bcd def